diff --git gcc/config.gcc gcc/config.gcc
index 6fcdd77..f570b33 100644
--- gcc/config.gcc
+++ gcc/config.gcc
@@ -2200,7 +2200,7 @@ i[34567]86-*-mingw* | x86_64-*-mingw*)
 			tmake_file="${tmake_file} i386/t-mingw-w32"
 			;;
 	esac
-        native_system_header_dir=/mingw/include
+        native_system_header_dir=/include
 	target_gtfiles="$target_gtfiles \$(srcdir)/config/i386/winnt.c"
 	extra_options="${extra_options} i386/cygming.opt i386/mingw.opt"
 	case ${target} in
diff --git libgomp/oacc-parallel.c libgomp/oacc-parallel.c
index c7e46e3..fe89892 100644
--- libgomp/oacc-parallel.c
+++ libgomp/oacc-parallel.c
@@ -31,7 +31,7 @@
 #include "gomp-constants.h"
 #include "oacc-int.h"
 #ifdef HAVE_INTTYPES_H
-# include <inttypes.h>  /* For PRIu64.  */
+# undef HAVE_INTTYPES_H
 #endif
 #include <string.h>
 #include <stdarg.h>
diff --git libgomp/target.c libgomp/target.c
index 3e292eb..288b4a3 100644
--- libgomp/target.c
+++ libgomp/target.c
@@ -33,7 +33,7 @@
 #include <stdbool.h>
 #include <stdlib.h>
 #ifdef HAVE_INTTYPES_H
-# include <inttypes.h>  /* For PRIu64.  */
+# undef HAVE_INTTYPES_H
 #endif
 #include <string.h>
 #include <assert.h>
